docs/release_notes/DRAFT_Release_notes_0.1.3.adoc

```diff
@@ -13,4 +13,8 @@ See the previous release notes for more details.
 == Features:
 
 == Fixes:
+- 406 error response from consors on get transactions with `Accept` set to any (\*/*)
+[quote]
+The requested formats in the Accept header entry are not matching the formats offered by the ASPSP"
```

Comment:
i think quotes at the end can be omitted

Reply:
yeah, removed